libyul/SideEffects.h

```diff
@@ -37,33 +51,48 @@ struct SideEffects
 	/// This means it cannot depend on storage or memory, cannot have any side-effects,
 	/// but it can depend on state that is constant across an EVM-call.
 	bool movable = true;
+	/// If true, the expression in this code can be moved or copied (together with its arguments)
+	/// across instructions and control flow branches whose 'effects' does not influence the
```

Comment:
```suggestion
	/// across control flow branches and instructions whose 'effects' does not influence the
```

Reply:
Or maybe: ... and instructions as long as these instruction's 'effects' do not...